data/I_80_test.json

```diff
@@ -1,11 +0,0 @@
-{"metadata": {"logFileName": "bsm.uper", "odeReceivedAt": "2017-09-26T20:00:08.48Z[UTC]", "payloadType": "us.dot.its.jpo.ode.model.OdeBsmPayload", "recordGeneratedBy": "OBU", "recordType": "bsmRxRecord", "recrodGeneratedAt": "2016-08-10T16:35:45.138Z[UTC]", "sanitized": false, "schemaVersion": 3, "serialId": {"bundleId": 12, "bundleSize": 1, "recordId": 5, "serialNumber": 0, "streamId": "f9c7635a-7763-4e21-bece-cb1104f143b9"}, "validSignature": true}, "payload": {"data": {"coreData": {"accelSet": {"accelYaw": 0.0}, "accuracy": {}, "brakes": {"abs": "unavailable", "auxBrakes": "unavailable", "brakeBoost": "unavailable", "scs": "unavailable", "traction": "unavailable", "wheelBrakes": {"leftFront": false, "leftRear": false, "rightFront": false, "rightRear": true, "unavailable": false}}, "heading": 52.8125, "id": "BEA10000", "msgCnt": 10, "position": {"elevation": 236.5, "latitude": 41.738136, "longitude": -106.587029}, "secMark": 18200, "size": {"length": 1000, "width": 30}, "speed": 7.02}, "partII": [{"id": "VEHICLESAFETYEXT", "value": {"pathHistory": {"crumbData": [{"elevationOffset": 9.5, "latOffset": 3.5e-06, "lonOffset": 0.0131071, "timeOffset": 33.2}, {"elevationOffset": 4.6, "latOffset": 7.4e-05, "lonOffset": 0.0131071, "timeOffset": 44.6}, {"elevationOffset": 3.5, "latOffset": 9.44e-05, "lonOffset": 5.1e-06, "timeOffset": 49.3}, {"elevationOffset": 204.7, "latOffset": 0.0001826, "lonOffset": 6.37e-05, "timeOffset": 71.7}, {"elevationOffset": 204.7, "latOffset": 0.0001313, "lonOffset": 6.06e-05, "timeOffset": 80.2}, {"elevationOffset": 204.7, "latOffset": 0.0001535, "lonOffset": 7.48e-05, "timeOffset": 92.9}, {"elevationOffset": 204.7, "latOffset": 0.000165, "lonOffset": 0.0001171, "timeOffset": 126.4}, {"elevationOffset": 204.7, "latOffset": 0.0001363, "lonOffset": 0.0001, "timeOffset": 136.1}, {"elevationOffset": 204.7, "latOffset": 0.0001196, "lonOffset": 0.0001028, "timeOffset": 141.9}, {"elevationOffset": 204.7, "latOffset": 0.0001111, "lonOffset": 9.8e-05, "timeOffset": 143.0}, {"elevationOffset": 204.7, "latOffset": 0.0131071, "lonOffset": 9.5e-06, "timeOffset": 348.0}, {"elevationOffset": 12.6, "latOffset": 0.0131071, "lonOffset": 0.0131071, "timeOffset": 437.8}, {"elevationOffset": 11.5, "latOffset": 0.0131071, "lonOffset": 0.0131071, "timeOffset": 468.2}, {"elevationOffset": 10.0, "latOffset": 0.0131071, "lonOffset": 0.0131071, "timeOffset": 474.0}, {"elevationOffset": 10.2, "latOffset": 0.0131071, "lonOffset": 7e-06, "timeOffset": 481.3}]}, "pathPrediction": {"confidence": 0.0, "radiusOfCurve": 0.0}}}]}, "dataType": "us.dot.its.jpo.ode.plugin.j2735.J2735Bsm"}, "schemaVersion": 3}
```

Comment:
question: was this file unused? or invalid format now?

Reply:
Invalid. No way of updating it without the original ASN.1